code/controllers/configuration/configuration.dm

```diff
@@ -231,7 +231,13 @@
 	return E.ValidateAndSet("[new_val]")
 
 /datum/controller/configuration/proc/LoadMOTD()
-	motd = file2text("[directory]/motd.txt")
+	var/list/motd_contents = list()
+
+	for (var/motd_file in CONFIG_GET(str_list/motd))
+		motd_contents += file2text("[directory]/[motd_file]")
```

Comment:
this should gracefully handle errors caused by missing files without breaking the motd. either using try/catch or just checking the file exists first.